### ISSUE TITLE
GALLERY-A in tekst

### ISSUE THREAD

**Issue by CatYo** [CatYo/for_me_gallery#1]
<!DOCTYPE html>
<html>
   <head>

<meta charset="utf-8"/>  
<link type="text/css" href="index.html" rel="stylesheet"/>
<meta name="viewport" content="width=device-width, initial-scale= 1.0">

   
   <script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-34943200-1', 'auto');
  ga('send', 'pageview');

</script>
    

      

```
   <style type="text/css">
```

.rubrika {
    font-family:Courier New, Courier, monospace;
    color:#9f9b74;
    background-color:darkslategray;
    text-align: left;
    font-size: 24px ;
    width: 100%;
    padding-left:10px;
    }

.gumbi{ 
    padding:0px;
    color: #9f9b74;
    } 

.tekst {
    color:#9f9b74;
    font-family:Courier New, Courier, monospace;
    text-align:justify;
    width:100%;
    max-width:600px;
    height:100%;
    float:right;
    position:relative;
    font-size: 18px;
   margin-right:-250px;
    margin-top:500px;
}

```
/* !!! */
```

 .tekst_izhod {font-size:14px;
    font-color:#9f9b74;
    font-family:Courier New, Courier, monospace;
    text-align:right;
    padding-right: 15px ;
    text-decoration: none;
     }

.potemnjeno {
    font-family:Courier New, Courier, monospace;
    color:#9f9b74;
    text-align:justify;
    }     

.body{
    background-color: #212930;
}           
# odebeljeno{

```
text-decoration-line: none;
font:bold;
```

}

img.responsive {
        width:100%;
        height:100%;
        }          

/\*  
.
.
.
.
SLIDESHOOOOW:
*/           

.rslides {
  position: relative;
  list-style:none;
  overflow:hidden;
  width: 100%;
  margin: 50px;
  padding-left: 50px;
  padding-bottom:0px;
    padding-top: 0px;
  }

.rslides li {
  -webkit-backface-visibility:hidden;
  position:absolute;
  display: none;
  max-width: 900px;
    width: 100%;
  left: 0;
  top: 0;
  }

.rslides li:first-child {
  position: relative;
  display: block;
  float: left;
  }

.rslides img {
  display: block;
  max-width:900px;
  width:100%;
  height: 100%;
  float: left;
  width: 100%;
  border: 0;
  }

 </style>

 </head> 

   <body >

  <div>
  <div>

```
<div class="rubrika"><b>ARCHIVE</b></div>



    <div class="gumbi"><p>
            <table align="right">
                <tr>
                    <td align="center"><a href="Archive_Colors.html"><img src="naslovne slike/button/red_button.jpg"  height="50" border="1px" weidth="50" /></a></td>      
                    <td align="center"><a href="Archive_WhiteInClearSpace.html"><img src="naslovne slike/button/pink_button.jpg" height="50" border="1px" weidth="50" /></a></td>
                    <td align="center"><a href="Archive_BlackInClearSpace.html"><img src="naslovne slike/button/blue_button.jpg" height="50" border="1px" weidth="50" /></a></td>
                    <td align="center"><a href="s.html"><img src="naslovne slike/button/green_button.jpg" height="50"   border="1px" weidth="50" /></a></td>
                </tr>
                <tr style="text-decoration:none; font-family:Courier New, Courier, monospace">
                    <td colspan="4">Enough of oldies and goldies?<a href="index.html" style="text-decoration:none; font-family:Courier New, Courier, monospace"><b style="text-decoration:none; font-family:Courier New, Courier, monospace; color:#9f9b74"> Click here.</b></a></td>
                </tr>
            </table></p>
        </div>

<div class="tekst">
                <table style="margin-top:-360px; margin-right: 100px "><tr><td><p style="font-size:30px"><b>WHITE IN CLEAR SPACE (2011)</b></p></td></tr>
                <tr><td><p>Opposite of Yin is Yang. It is represented by a woman in a white space. At first, she is dressed in black, 
```

which symbolizes a man, her opposite. Then she changes into a bright, colorful kerchief; she surrounds herself with verdure. 
She holds a parrot hairpin between her teeth, which symbolizes her desire to free herself from the shackles, 
the desire for happiness, harmony, and love. However, a force of a man keeps her grounded, therefore she cries. 
Due to the syllables, the symbolism of a parrot represents the reconciliation on account of unrealizable desire 
to be something more. In Chinese culture the Yang represents a male part of the Yin – Yang symbol.</p>  
                   </td></tr></table>
  </div> 

```
<div>
     <ul class="rslides" style="list-style: none;  padding-left:100px; padding-top:170px" >
          <li><img src="http://www.visitsinj.com/Content/images/download/sinjsko-polje-900.jpg" alt="" ></li>
          <li><img src="http://www.visitsinj.com/Content/images/download/sinjsko-polje-900.jpg" alt=""></li>
          <li><img src="http://www.visitsinj.com/Content/images/download/sinjsko-polje-900.jpg" alt=""></li>

        </ul>
     </div>     
```

   </div> 
   

<script src="http://ajax.googleapis.com/ajax/libs/jquery/1.8.3/jquery.min.js"></script>

<script src="responsiveslides.min.js"></script> 

<script>
  $(function() {
    $(".rslides").responsiveSlides({
  auto: true,             // Boolean: Animate automatically, true or false
  speed: 500,            // Integer: Speed of the transition, in milliseconds
  timeout: 4000,          // Integer: Time between slide transitions, in milliseconds
  pager: false,           // Boolean: Show pager, true or false
  nav: false,             // Boolean: Show navigation, true or false
  random: false,          // Boolean: Randomize the order of the slides, true or false
  pause: false,           // Boolean: Pause on hover, true or false
  pauseControls: true,    // Boolean: Pause when hovering controls, true or false
  prevText: "Previous",   // String: Text for the "previous" button
  nextText: "Next",       // String: Text for the "next" button
  maxwidth: "",           // Integer: Max-width of the slideshow, in pixels
  navContainer: "",       // Selector: Where controls should be appended to, default is after the 'ul'
  manualControls: "",     // Selector: Declare custom pager navigation
  namespace: "rslides",   // String: Change the default namespace used
  before: function(){},   // Function: Before callback
  after: function(){}     // Function: After callback
});
  });
</script>

</body>
</html
